lib/cli-engine/file-enumerator.js

```diff
@@ -217,6 +217,14 @@ class FileEnumerator {
         });
     }
 
+    /**
+     * The `RegExp` object that tests if a file path has the allowed file extensions.
+     * @type {RegExp}
+     */
+    get extRegExp() {
```

Comment:
I don't have a strong preference here, but now that this is exposed to other parts of ESLint, I think we could use a longer and more clear property name (e.g., `extensionRegExp`).